src/data/amd_genoa_ctrs.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "stall_backend_pkc",
+    "nrs": [
+      {
+        "perf_type": "RAW",
```

Comment:
Best I can tell, `"perf_type": "RAW"` can be inferred by the presence of the `"config"` field.  Everything is either raw or has nrs.

Reply:
RAW here indicates CPU. We're looking to add support for other types of PMUs too which the perf_type will hold.